src/components/PeopleTable.tsx

```diff
@@ -10,44 +57,72 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
-              <a href="#/people?sort=name">
+              <SearchLink
+                params={{ sort: 'name' }}
+                onClick={e => {
```

Comment:
instead of changing params manually on click you should just set the correct params for link
You can use function to get correct params (including order) based on current searchParams values

That's the way we want to work with Router links 🙂 So after that remove onClick handler at all

Also, use Search link in PeopleFilters